src/google/appengine/ext/ndb/polymodel.py

```diff
@@ -193,7 +193,7 @@ def _update_kind_map(cls):
     cls._kind_map[cls._class_name()] = cls
     class_key = cls._class_key()
     if class_key:
-      cls._class_map[tuple(class_key)] = cls
+      cls._class_map[tuple(map(six.ensure_binary, class_key))] = cls
```

Comment:
I've gone with `six.ensure_binary` because that's what `_class_key` was using.

Given the library isn't PY2-compatible and assuming `_class_key` always returns a list of `str`, a lambda should be faster:
```suggestion
      cls._class_map[tuple(map(lambda c: c.encode(), class_key))] = cls
```